packages/api/src/routes/locationRoute.js

```diff
@@ -64,6 +64,9 @@ router.post('/barn', hasFarmAccess({ body: 'farm_id' }), checkScope(['add:barn']
 router.post('/field', hasFarmAccess({ body: 'farm_id' }), checkScope(['add:fields']),
   modelMapping['field'], createLocation('field'));
 
+router.post('/farm_site_boundary', hasFarmAccess({ body: 'farm_id' }), checkScope(['edit:farm_site_boundary']),
```

Comment:
Shouldn't the permission checker check for `add:farm_site_boundary` instead of `edit`?

Reply:
completely right